app/controllers/accounts_controller.rb

```diff
@@ -132,29 +113,39 @@ def move_account(format, target_group)
   end
 
   def group
-    @group ||= team.groups.find(params[:group_id])
+    @group ||= @account.group
   end
 
-  def accounts_breadcrumbs
-    add_breadcrumb t('teams.title'), :teams_path
-    add_breadcrumb team.label, :team_groups_path
+  def team
+    @team ||= group.team
+  end
 
-    add_breadcrumb @group.label if action_name == 'index'
+  def account
+    @account ||= Account.find(params[:id])
+  end
+
+  def accounts_breadcrumbs
+    teams_breadcrumbs
 
-    if action_name == 'show' || action_name == 'edit'
-      add_breadcrumb @group.label, :team_group_accounts_path
-      add_breadcrumb @account.label
+    if %w[show edit new].include?(action_name)
```

Comment:
new ?
you might show breadcrumbs on new ... but then you have to use another label Line 132
something like "Account"